website/content/api-docs/index.mdx

```diff
@@ -75,6 +75,29 @@ $ curl \
     http://127.0.0.1:8500/v1/kv/foo
 ```
 
+## Accepts URL-Encoded Resource Names
```

Comment:
```suggestion
## URL-Encoded Resource Names
```
Maybe it's just me, but the verb at the beginning of this heading was jarring, especially considering that all (or most) of the other headings in this section have a noun-phrase. Ignore if I'm the odd man out on this one.

Reply:
I agree. I felt weird about this when I wrote it, too.